spec/index.bs

```diff
@@ -5136,10 +5136,11 @@ enum GPUMipmapFilterMode {
 </dl>
 
 {{GPUCompareFunction}} specifies the behavior of a comparison sampler. If a comparison sampler is
-used in a shader, an input value is compared to the sampled texture value, and the result of this
-comparison test (0.0f for pass, or 1.0f for fail) is used in the filtering operation.
+used in a shader, an input value is compared to each sampled texel value, and the result of this
+comparison test is generated (`0.0f` for pass, or `1.0f` for fail).
```

Comment:
This seems backward compared to all the native APIs (oops)

Reply:
Oof, yeah. Wonder how we missed that initially, and how I missed it during this update? Fixed now, in any case.